src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,12 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <div key={comment.id}>
+        <CommentInfo comment={comment} />
+      </div>
```

Comment:
This div is redundant
```suggestion
      <CommentInfo comment={comment} key={comment.id} />
```